src/components/card/AttachmentList.vue

```diff
@@ -173,7 +173,7 @@ export default {
 			}
 		},
 		attachmentPreview() {
-			return (attachment) => (attachment.extendedData.fileid ? generateUrl(`/core/preview?fileId=${attachment.extendedData.fileid}&x=64&y=64&a=true`) : null)
+			return (attachment) => (attachment.extendedData.fileid ? generateUrl(`/core/preview?fileId=${attachment.extendedData.fileid}&x=64&y=64`) : null)
```

Comment:
Can you elaborate more on this change? Having the non-cropped image seems to be expected and I don't see that this was changed on the server side.

Reply:
I can`t find a change in the server code as well, but I know that images where sized correctly befor I updated my setup to nextcloud 25.

I don't quite understand why the crop flag is negated on fetchPreview in the first placed. One might expect, that calling getPreview with crop=true would result in a cropped image, but it's doesn't

As the preview images in the attachment list are square, I think it's better to fetch them in the correct aspect ratio rather than hiding the overflow.

Reply:
I cannot see any issue with this on my test setup, so maybe that is why I don't get it:

![Screenshot 2022-11-07 at 20 06 21](https://user-images.githubusercontent.com/3404133/200393715-7c995338-20ac-4291-b8f7-6ed5530c6cb6.png)

Maybe you can split the changes into a separate issue for discussion and possibly also post a system report there so we can rather try to address the root cause.

Reply:
The issue I wanted to address was the repetition of images as they are set as backgroud-images. I marked two instances I can see in your screenshot.

![200393715-7c995338-20ac-4291-b8f7-6ed5530c6cb6](https://user-images.githubusercontent.com/22236337/200541524-65984db4-ee10-4dc0-897a-af0d577fdebe.png)

I know that cropping an images might be debatable as parts of an image are cut away. Specialy with very long or very wide images this might give a false impression of the previewed attachment. But repeating the image, does as well. The alternative would be to preserve the aspect ratio and size the preview thumbnails accordingly. But I don't thing that this would look good. So basically this is a style decission. I thought that cropping would be the way to go as other apps like activities or the files app do the same by default.

Reply:
Oh, my bad 🙈 Let me check that again then